src/process.cr

```diff
@@ -414,6 +414,27 @@ class Process
   private def close_io(io)
     io.close if io
   end
+
+  # Changes the root directory and the current working directory for the current
+  # process.
+  #
+  # ```
+  # Process.chroot("/var/empty")
+  # ```
+  def self.chroot(path : String) : Nil
+    path.check_no_null_byte
+    if LibC.chroot(path) != 0
+      raise Errno.new("Failed to chroot")
+    end
+
+    if LibC.chdir("/") != 0
+      errno = Errno.new("chdir after chroot failed")
+      errno.callstack = CallStack.new
+      errno.inspect_with_backtrace(STDERR)
+      STDERR.puts "Unresolvable state, exiting..."
+      Process.exit(1)
```

Comment:
Shouldn't this use `abort("Unresolvable state, exiting...")` instead of `STDERR.puts + Process.exit`?

Reply:
Since #5413 is merged, it should also set `AtExitHandlers.exception = errno`. With this in place `errno.inspect_with_backtrace(STDERR)` is needless.

Reply:
`AtExitHandlers.exception` prints `Unhandled exception` but it is not - and you just can't handle this exception.

Reply:
But `abort` would be great.